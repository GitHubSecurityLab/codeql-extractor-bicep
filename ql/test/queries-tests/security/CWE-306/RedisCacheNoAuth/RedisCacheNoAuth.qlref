security/CWE-306/RedisCacheNoAuth.ql
