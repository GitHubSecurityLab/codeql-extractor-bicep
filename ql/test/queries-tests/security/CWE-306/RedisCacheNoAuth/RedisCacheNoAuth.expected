| app.bicep:26:1:34:1 | RedisCacheResource | Redis Cache 'noauth-redis' has authentication disabled, allowing unauthenticated access. |
